### DISCUSSION TITLE
Leave Blank v21.02.0-rc3 Fixed

### DISCUSSION THREAD

**Discussion by tommiefr** [nicholas-opensource/OpenWrt-Autobuild#16] [General]
Just installed this, but on several pages I get (in Chrome):
ERR_SSL_VERSION_OR_CIPHER_MISMATCH

How to solve this?

**Comment by nicholas-opensource** [comment]
osi layer 3 router has nothing to do with your issue. If you used proxy or sslvpn, check the availability and settings of your certificate.

or:
https://kinsta.com/knowledgebase/err_ssl_version_or_cipher_mismatch/

**Comment by tommiefr** [comment]
It is not only in chrome I am getting that error. Other browser as well...
I have an VPN, but same error after closing it.
It is on some pages, not all. F.e nu.nl

I like your build, because the full upload speed I get on r2s. Even on VPN I get 160/160!
But this error I must solve to use it further.

I tried already your link, but I have windows 10 and last version of chrome. It looks like a package is missing but I cannot find what.

**Reply by nicholas-opensource** [reply]
https://www.ssllabs.com/ssltest/analyze.html?d=nu.nl&s=184.25.254.56 

nu.nl works fine under my network and it can pass ssllabs test. Maybe you can clean up SSL state in previous links

**Reply by tommiefr** [reply]
Ok really strange....I cleaned in internet settings the SSL state....
What else can I do?

**Reply by nicholas-opensource** [reply]
You can try to use the official openwrt firmware to troubleshoot the problem. I use libstream-OpenSSL to replace wolfssl, may cause problems but the probability is extremely low.

**Reply by tommiefr** [reply]
I went back to this version and here it is working. Only the up- and download speeds are not that high. Furthermore rock-stable.

https://github.com/anaelorlinski/OpenWrt-NanoPi-R2S-R4S-Builds

libustream-wolfssl20201210 is installed in that version as well and nu.nl is working. 
When you have more ideas how to solve this, I really like to hear!

**Reply by nicholas-opensource** [reply]
I will test wolfssl these days and compare libstream.

**Comment by nicholas-opensource** [comment]
But this brings me to another possibility:
DNS hijacking or DNS errors will cause resolution to other websites.

**Reply by tommiefr** [reply]
I use 1.1.1.1 and 1.0.0.1 as custom dns

**Reply by tommiefr** [reply]
Can it be that it's not compatible with cloudfare?

**Reply by nicholas-opensource** [reply]
libstreams-wolfssl and OpenSSL are both working fine with nu.nl
Try to use `nslookup nu.nl 1.1.1.1` or `dig nu.nl @1.1.1.1` to check whether to return correct IP

**Reply by tommiefr** [reply]
Re installing the latest version now. Will test

**Reply by tommiefr** [reply]
Don't know what was setup wrong but after starting all over again, everything works great!!
Thanks!!
